arbeitszeiterfassung_s4a/arbeitszeiterfassung_s4a/doctype/working_time/working_time.py

```diff
@@ -7,12 +7,10 @@
 from frappe import _
 from frappe.model.document import Document
 
-HALF_DAY = 3.25
-OVERTIME_FACTOR = 1.15
-MAX_HALF_DAY = HALF_DAY * OVERTIME_FACTOR * 60 * 60
 FIVE_MINUTES = 5 * 60
 ONE_HOUR = 60 * 60
 
+default_activity = frappe.db.get_single_value('Working Time Settings', 'default_activity')
```

Comment:
Das sollte keine globale Variable sein. Besser in eine Funktion `get_default_activity` verpacken:

```py
def get_default_activity():
	return frappe.db.get_single_value("Working Time Settings", "default_activity")
```

Und bitte auf eine einheitliche Formatierung achten (doppelte Anführungszeichen)